### ISSUE TITLE
No writable property 'url' on class: io.kamax.mxisd.config.ListenerConfig

### ISSUE THREAD

**Issue by NullIsNot0** [kamax-matrix/mxisd#131]
After migrating from 1.2.2 to 1.3.0 e-mail notification configuration crashes mxisd docker container.

Configuration part:
```
...
matrix:
    domain: example.com
    listener:
        localpart: appservice-mxisd
        token:
            as: 9pBE4fo8qsFjOM0dELtpvTrA6BLYCndS8Nzqrv1laTqiW3oAXDHzxQ0otT3p1J1j
            hs: yNabQ67LO55IedMod2cQYwaEUfWtpEsAKz4bNS2sQ8SCr8NyOX7y9rFv1kDvxFja
        url: http://matrix-synapse:8090
...
```

Error I get:
```
No writable property 'url' on class: io.kamax.mxisd.config.ListenerConfig
in 'reader', line 37, column 14:
url: http://matrix-synapse:8090
```
This configuration worked well in 1.2.2.

**Comment by maxidorius** [comment]
Could you give `kamax/mxisd:1.3.0-7-g99697d7` a try please?

**Comment by NullIsNot0** [comment]
Just tried `kamax/mxisd:1.3.0-7-g99697d7`. It works well. No more errors, and functionality works. Thank you a lot!

**Comment by maxidorius** [comment]
Awesome! Thank you for reporting and testing. Please leave this issue open for a few days so people who are hit by it can find it easily.

For anyone using the Debian package, you'll find a package for this fix [here](https://matrix.kamax.io/download/mxisd_1.3.0-7-g99697d7_all.deb)

**Comment by maxidorius** [comment]
Fix is part of the [v1.3.1 release](https://github.com/kamax-matrix/mxisd/releases/tag/v1.3.1)
